src/autocomplete/AutocompleteView.js

```diff
@@ -20,12 +20,15 @@ type Props = $ReadOnly<{|
   text: string,
   selection: InputSelection,
   onAutocomplete: (input: string) => void,
+  processAutoComplete: (completion: string, completionType: string) => void,
```

Comment:
Having two different callbacks here for the same event doesn't feel like the right interface. Better to adjust the one callback's interface to do what we need it to.

Reply:
Modified the current callback